assets/sass/protocol/includes/mixins/_gradients.scss

```diff
@@ -1,54 +1,54 @@
+/* stylelint-disable declaration-block-no-duplicate-properties */
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-
 // Horizontal gradient, from left to right.
 // Creates two color stops, start and end, by specifying a color and position for each color stop.
 @mixin gradient-horizontal($start-color: #555, $start-percent: 0%, $end-color: #333, $end-percent: 100%) {
-  background-image: -webkit-linear-gradient(left, $start-color $start-percent, $end-color $end-percent); // Safari 5.1-6, Chrome 10+
-  background-image:         linear-gradient(to right, $start-color $start-percent, $end-color $end-percent); // Standard, IE10, Firefox 16+, Opera 12.10+, Safari 7+, Chrome 26+
+    background-image: linear-gradient(to left, $start-color $start-percent, $end-color $end-percent); // Safari 5.1-6, Chrome 10+
+    background-image: linear-gradient(to right, $start-color $start-percent, $end-color $end-percent); // Standard, IE10, Firefox 16+, Opera 12.10+, Safari 7+, Chrome 26+
```

Comment:
Here's an example of where we have duplicate `background-image` properties, but the first property is using an invalid (non-standards based) syntax, because it used to be prefixed with `-webkit`. I think we can remove the first property and just keep the second.

There are a few more occurrences below